python/src/gluon_ir.cc

```diff
@@ -698,6 +698,14 @@ void init_gluon_ir(py::module &&m) {
              self.create<ttag::BufferStoreOp>(storedValue, ptr, offsets,
                                               Value() /*stride*/, cache, mask);
            })
+      .def("create_buffer_atomic_rmw",
+           [](GluonOpBuilder &self, tt::RMWOp op, Value ptr, Value offsets,
+              Value mask, Value value, tt::MemSemantic sem,
```

Comment:
Nit: maybe put `value` before `mask` to be consistent with L707 and the call sites of this function?